spec/knapsack_pro/slow_test_file_determiner_spec.rb

```diff
@@ -67,7 +67,7 @@
 
     context 'when json report does not exist' do
       it do
-        expect { subject }.to raise_error(RuntimeError, 'Report with slow test files was not generated yet. If you have enabled split by test cases https://github.com/KnapsackPro/knapsack_pro-ruby#split-test-files-by-test-cases and you see this error it means that your tests accidentally cleaned up tmp/knapsack_pro directory. Please do not remove this directory during tests runtime!')
+        expect { subject }.to raise_error(RuntimeError, 'Report with slow test files was not generated yet. If you have enabled split by test cases https://github.com/KnapsackPro/knapsack_pro-ruby#split-test-files-by-test-cases and you see this error it means that your tests accidentally cleaned up the .knapsack_pro directory. Please do not remove this directory during tests runtime!')
```

Comment:
gramatically, this should read "has not been generated yet"

I would also say either "_The_ report" or "_A_ report" depending on what makes more sense.